iphone/Classes/APIModule.m

```diff
@@ -91,34 +72,38 @@ - (void)error:(NSArray *)args
   [self logMessage:args severity:@"error"];
 }
 
-- (void)trace:(NSArray *)args
+- (void)trace:(id)args
 {
   [self logMessage:args severity:@"trace"];
 }
 
-- (void)timestamp:(NSArray *)args
+- (void)timestamp:(id)args
 {
-  NSLog(@"[TIMESTAMP] %f %@", [NSDate timeIntervalSinceReferenceDate], [self transform:[args objectAtIndex:0]]);
+  id firstArg = args;
+  if ([args isKindOfClass:[NSArray class]]) {
+    firstArg = [args objectAtIndex:0];
+  }
```

Comment:
I'm not sure if that is still true now with the new API. I'm pretty sure I *had* to change this as a result of finding that the new API attempts to do it's own argument transforms to target types, and that if a method truly accepts multiple types you need to move to `id` and do the type-checking yourself. It's been a while since I worked on this, but I believe it tried to coerce arguments to the native type declared, and would fail to turn a String or Number into an Array.